turbinia/client.py

```diff
@@ -439,6 +439,7 @@ def __init__(self, *_, **__):
   def start(self):
     """Start Turbinia Celery Worker."""
     log.info('Running Turbinia Celery Worker.')
+    self.worker.task(task_manager.task_runner, name="task_runner")
```

Comment:
Small nit, but can we keep the quote characters as single quotes to be consistent?  I think there are a couple other instances as well.